uint8 PID=0
uint8 PWM=1
uint8 control_mode
int32 left_mower_speed
int32 right_mower_speed